spec/workflow/intrinsic_function_spec.rb

```diff
@@ -583,14 +613,10 @@
         expect(result).to eq("[\"foo\",\"bar\"]")
       end
 
-      it "requires 1 parameter" do
+      it "fails with wrong number of arguments" do
         expect { described_class.value("States.JsonToString()", {}, {}) }.to raise_error(ArgumentError, "wrong number of arguments to States.JsonToString (given 0, expected 1)")
         expect { described_class.value("States.JsonToString($.input, true)", {}, {"input" => 5}) }.to raise_error(ArgumentError, "wrong number of arguments to States.JsonToString (given 2, expected 1)")
       end
-
-      it "requires a valid path" do
```

Comment:
I dropped this test (and the other similar one), because we have a whole separate section in the tests for jsonpath failures making this redundant.